www/css/index.css

```
@@ -7,14 +15,18 @@
   font-weight: bold;
   font-style: normal;
 }
+
 html, body {
   margin: 0;
   padding: 0;
+  font-family: "PT Sans", -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+  text-align: center;
 }
+
 .container {
   height: 100vh;
   width: 100vw;
-  padding: 1em;
+  padding: 4em;
```

```suggestion
  padding: 4em 2em;
```

Pour que la marge ne soit pas trop important en mobile (déjà que le texte sera bien gros)